Frends.Csv.Tests/Tests.cs

```diff
@@ -133,13 +134,105 @@ public void TestParseWillAllKindOfDataTypes()
             Assert.That(itemArray[6], Is.EqualTo('f'));
 
             Assert.That(itemArray[7].GetType(), Is.EqualTo(typeof(DateTime)));
-            Assert.That(itemArray[7], Is.EqualTo(new DateTime(2008,9,15)));
+            Assert.That(itemArray[7], Is.EqualTo(new DateTime(2008, 9, 15)));
 
             Assert.That(itemArray[8].GetType(), Is.EqualTo(typeof(DateTime)));
-            Assert.That(itemArray[8], Is.EqualTo(new DateTime(2008,5,1,10,34,42)));
+            Assert.That(itemArray[8], Is.EqualTo(new DateTime(2008, 5, 1, 10, 34, 42)));
+        }
+
+        [Test]
+        public void TestParseTreatMissingFieldsAsNullSetToTrue()
+        {
+            var csv =
+                @"header1,header2,header3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2
+                value1,value2,value3
+                value1,value2,value3";
+
+            var result = Csv.Parse(new ParseInput()
+            {
+                ColumnSpecifications = new ColumnSpecification[0],
+                Delimiter = ",",
+                Csv = csv
+            }, new ParseOption() { ContainsHeaderRow = true, CultureInfo = "fi-FI", TreatMissingFieldsAsNulls = true });
+            var resultJson = (JArray)result.ToJson();
+            Assert.That(resultJson[6].Value<string>("header3"), Is.EqualTo(null));
+
+            var resultXml = result.ToXml();
+            Assert.That(resultXml, Does.Contain("<header3 />"));
+
+            var resultData = result.Data;
+            var nullItem = resultData[6][2];
+
+            Assert.That(nullItem, Is.EqualTo(null));
         }
 
+        [Test]
+        public void TestParseTreatMissingFieldsAsNullDefault()
+        {
+            var csv =
+                @"header1,header2,header3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2,value3
+                value1,value2
+                value1,value2,value3
+                value1,value2,value3";
+            try
+            {
+                var result = Csv.Parse(new ParseInput()
+                {
+                    ColumnSpecifications = new ColumnSpecification[0],
+                    Delimiter = ",",
+                    Csv = csv
+                }, new ParseOption() { ContainsHeaderRow = true, CultureInfo = "fi-FI" });
```

Comment:
This does not have to be the whole test, you can just test the constructor if this was important. However the default value attribute does not really affect the task execution in unit tests, so this is really not testing much. I would suggest just to remove this and test directly in Frends, as for testing this functionality we need Frends involvement.

One this you can do however is to check that the default value attribute on the property is set to false, which will affect how Frends interprets this.